core/history.go

```diff
@@ -42,16 +43,17 @@ func (mw *Middleware) HistoryShowVersion(cctx context.Context, req *pb.RpcHistor
 		if err != nil {
 			return fmt.Errorf("resolve spaceID: %w", err)
 		}
-		obj, ver, err = hs.Show(domain.FullID{
+		id := domain.FullID{
 			SpaceID:  spaceID,
 			ObjectID: req.ObjectId,
-		}, req.VersionId)
+		}
+		obj, ver, err = hs.Show(id, req.VersionId)
 		return
 	}); err != nil {
-		return response(nil, nil, err)
+		return response(nil, nil, nil, err)
 	}
 
-	return response(obj, ver, nil)
+	return response(obj, ver, blocksModifiers, nil)
```

Comment:
blocksParticipants would always be nil right? you don't set them anywhere, pls fix that

Reply:
I guess you should call the method there

Reply:
Fixed